doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -2066,3 +2066,32 @@ Example of Modbus logging of a request and response:
       "error_flags": "DATA_VALUE",
     },
   }
+
```

Comment:
What does Suricata interpret as a transaction for QUIC ?
Is it something unidirectional ? Or something with request and response ?

Reply:
Current implementation is unidirectional, this can be improved in the future

Reply:
No problem with unidirectional.

Just wondering : does it make sense for QUIC to have bidirectional transaction ? Is there a concept of request and response ?